### ISSUE TITLE
When I build my application with esbuild and docker I get: file not found _tiktoken_bg.wasm

### ISSUE THREAD

**Issue by NoodleOfDeath** [dqbd/tiktoken#6]
what is _tiktoken_bg.wasm? and why is my docker image trying to open this? how do i get this to work?

**Comment by NoodleOfDeath** [comment]
Currently, I am trying a work around in my Docker file `touch dist/_tiktoken.wasm`

**Comment by NoodleOfDeath** [comment]
nope... still broken what the heck

**Comment by NoodleOfDeath** [comment]
when did this start becoming an issue? this was never a problem before?

**Comment by NoodleOfDeath** [comment]
I had to add it to my Dockerfile after esbuild. Seems like a bundler issue? 

```
RUN pnpm build
COPY _tiktoken_bg.wasm ./dist
CMD pnpm start
```

**Comment by dqbd** [comment]
It seems like the issue has been resolved by upgrading to 4.0 version, where bundler issues were tackled. Closing this issue for now.

**Comment by NoodleOfDeath** [comment]
@dqbd You da man!
